src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -56,18 +69,18 @@ It mainly consists of two key components: the IP Address Manager and Globalnet.
 The IP Address Manager (IPAM) component does the following:
 
 * Creates a pool of IP addresses based on the `GlobalCIDR` configured on cluster.
-* Allocates a globalIp from the GlobalIp pool on creation of a Pod, Service, or ServiceExport.
-* Annotates the Pod or exported Service with `submariner.io/globalIp=<global-ip>`.
-* On deletion of a Pod, Service, or ServiceExport, releases its globalIp back to the pool.
+* Allocates a globalIp from the GlobalIp pool on creation of ClusterGlobalEgressIPs, GlobalEgressIPs or GlobalIngressIPs.
+* On deletion of a ClusterGlobalEgressIPs, GlobalEgressIPs or GlobalIngressIPs, releases its globalIp back to the pool.
 
 #### Globalnet
 
 This component is responsible for programming the routing entries, iptable rules and does the following:
 
 * Creates initial iptables chains for Globalnet rules.
-* Whenever a Pod is annotated with a globalIp, creates an egress SNAT rule to convert the source Ip from the Pod's Ip to the Pod's globalIp
+* Whenever GlobalEgressIPs are allocated, creates corresponding SNAT rules to convert source Ip from Pod's Ip to the corresponding globalIp.
```

Comment:
```suggestion
* Whenever GlobalEgressIPs are allocated, creates corresponding SNAT rules to convert source Ip for all the matching pods to the corresponding globalIP allocated to GlobalEgressIP object..
```

Reply:
We really don't have an  explicit "Globalnet" component (anymore) but I guess it's OK 😄 

```suggestion
* For each `GlobalEgressIP`, creates corresponding SNAT rules to convert the source IPs for all the matching pods to the corresponding global IP(s) allocated to the `GlobalEgressIP` object.
```